index.html

```diff
@@ -6,15 +6,128 @@
+        <div class="container">
+            <div class="row">  
+                <div class="col-3">
+                    <h3>ALLAN JOHNSON</h3>
+                    <p>Personal Life Coach</p>
+                </div>
+                    <div class="col-6">
+                     <nav class="navigation-bar">
+                        <ul>
+                         <li><a href="#">MENU</a>
+                              <ul class="top-menu">
+                                <li><a href="#">About</a></li>
+                                <li><a href="#">Services</a></li>
+                                <li><a href="#">Motivation Reading</a></li>
+                                <li><a href="#">Blog</a></li>
+                                <li><a href="#">Contact</a></li>
+                              </ul>
+                         </li>
+                        </ul>
+                     </nav>
+                    </div>
+                  <div class="col-3">
+                    <div class="log-in">
+                    <i class="fa fa-user">  Log in</i>
+                  </div>   
+                </div> 
+            </div>
+        </div>             
+    </header>
+                           <!-- hero .big picture -->
+            
+                    <div class="container">
+                        <div class="row">
+                            <div class="col-12">
+                                <h5>AMBITION THE FIRST STEP TOWARDS</h5>
+                                <h1>SUCCESS</h1>
+                                <button class="book-now">Book Now</button>
+                            <img src="imgs/image3.jpg" alt="landscape" width="100%">
+                        </div>
+                     </div>
+                     <!-- hero ended -->
+                        <div class="row">
+                          <div class="col-6">
+                             <h2>About Me</h2>
+                              <p>
+                                SMS allows being in ongoing contact with,them and benefits
+                                and that and action.SMS also has agreat deal of value.
+                                an alert enhance service.has been providing  solutions to
+                                for over 18 years.It services
+                                more than 18,000 over 5 Billion SMS texts per.
+                              </p>
+                              <button class="read-more">Read More</button>
+                        </div>
+                           <div class="col-6">
+                              <img src="imgs/image-coach.jpg" alt="personal coach" width="100%">
+                            </div>
+                        </div>
 
-    <!-- Add your markup here -->
-
+                        <div class="row">
+                            <div class="col-12">
+                            <img src="imgs/image5.jpg" alt="sea landscape" width="100%">
+                          <!-- </div>
```

Comment:
If you are not keeping this, then delete. Use comments onlu to communicate something to the reviewer otherwise remove